cylc/rose/stem.py

```diff
@@ -17,24 +17,20 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
-"""rose stem [path]
+"""rose stem [path] [options]
```

Comment:
To match other commands this should be "rose stem [OPTIONS] [PATH]"

Reply:
Done